modules/cli/src/main/scala/scala/cli/commands/util/PackageOptionsUtil.scala

```diff
@@ -92,6 +92,9 @@ object PackageOptionsUtil {
         internal = baseOptions.internal.copy(
           // computing the provided modules sub-graph needs the final Resolution instance
           keepResolution = provided.nonEmpty
+        ),
+        internalDependencies = baseOptions.internalDependencies.copy(
+          addRunnerDependencyOpt = Some(false)
```

Comment:
Why was `addRunnerDependencyOpt` added to classPath by default before? In some cases it is needed?

Reply:
We use it in the `run` sub-command mainly. It gives better stack-traces in particular. But I wouldn't add it by default in package (where as a user, I'd expect the main class I provide to be used as is, rather than the one of the runner being used).